fdbserver/BackupWorker.actor.cpp

```diff
@@ -938,7 +932,6 @@ ACTOR Future<Void> uploadData(BackupData* self) {
 		}
 
 		if (self->allMessageSaved()) {
-			self->eraseMessages(self->messages.size());
 			return Void();
```

Comment:
is it related the first bulletin you mentioned, `Fixed memory bookkeeping issue in Backup Worker. Previously it didn't release flow lock correctly when erasing messages.`.  Why does it work by skipping erasing messages?

Reply:
It's a fix for a simulation test failure. If allMessageSaved() is true, the backup worker has done its job and is going to terminate soon. Its fields are going to be destructed so no need to call self->eraseMessages() to release the memory.  If we call self->eraseMessages() here, it'll cause a very tricky race condition of FlowLock.